quetz/tests/test_cli.py

```diff
@@ -86,9 +86,8 @@ def get_db(_):
     with mock.patch("quetz.cli.get_session", get_db):
         cli.create(
             Path(config_dir) / "new-deployment",
-            config_file_name="config.toml",
             copy_conf="config.toml",
-            create_conf=None,
+            create_conf=False,
             dev=True,
         )
 
```

Comment:
it would be cool to add some unit tests. cli module is one of the least tested, according to codecov: https://codecov.io/gh/mamba-org/quetz/tree/master/quetz